pkg/volume/glusterfs/glusterfs.go

```diff
@@ -681,7 +691,16 @@ func (p *glusterfsVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 		glog.V(4).Infof("glusterfs: not able to parse your claim Selector")
 		return nil, fmt.Errorf("glusterfs: not able to parse your claim Selector")
 	}
-	glog.V(4).Infof("glusterfs: Provison VolumeOptions %v", p.options)
+
+	recyclePolicy := string(api.PersistentVolumeReclaimRecycle)
+	requestedPolicy := string(p.options.PersistentVolumeReclaimPolicy)
```

Comment:
the PV controller will never pass you Recycle because sc.reclaimPolicy==Recycle is invalid, so no need to worry about this case 👍